test/e2e/lib/components/site-editor-component.js

```diff
@@ -156,4 +156,19 @@ export default class SiteEditorComponent extends AsyncBaseContainer {
 			)
 		);
 	}
+
+	async toggleNavigationSidebar() {
+		await driverHelper.clickWhenClickable(
+			this.driver,
+			By.css( '.edit-site-navigation-toggle__button' )
+		);
+	}
+
+	/**
+	 * Alias for `toggleNavigationSidebar`.
+	 * We need to have the same function names in both
+	 * this and gutenberg editor component to
+	 * make sure general tests are working as expected.
+	 */
```

Comment:
`tip`: If you're using VSCode, there's a neat extension that wraps the comments automatically for you: https://marketplace.visualstudio.com/items?itemName=stkb.rewrap. I've set it up to wrap at 100th column as it's what we've set for Prettier.

Reply:
I just installed this on your suggestion and its great!